docs/source/features/version_control/branch.rst

```diff
@@ -0,0 +1,105 @@
+..
+ Copyright 2021 Graviti. Licensed under MIT License.
+ 
+########
+ Branch
+########
+
+Each dataset is created with a default branch ``main``. When getting a dataset through the SDK,
+its version is the latest commit of the default branch.
+
+The following sections will introduce the operations and precautions related to branches
+in the SDK. Of course, it is necessary to get a dataset first:
+
+.. code:: python
+
+   from graviti import Workspace
+   ws = Workspace(f"{YOUR_ACCESSKEY}")
+   dataset = ws.datasets.get(f"{DATASET_NAME}")
+
+.. note::
+   In most cases, a branch can be thought of as a named commit. But if there is a dataset that
+   has just been created and has no commit history, the commit id of its default branch is
+   ``None``.
+
+
+*****************
+ Create a Branch
+*****************
+
+SDK provides method :meth:`~graviti.manager.branch.BranchManager.create` to support creating a
+branch based on a commit:
+
+.. code:: python
+
+   dataset.branches.create(f"{BRANCH_NAME}")
+   dataset.branches.create(f"{BRANCH_NAME}", f"{BRANCH_NAME}")
+   dataset.branches.create(f"{BRANCH_NAME}", f"{COMMIT_ID}")
+   dataset.branches.create(f"{BRANCH_NAME}", f"{TAG_NAME}")
+
+.. note::
+   If no revision is specified, the created branch will be based on the current commit of the
+   dataset, which can be viewed by ``dataset.HEAD.commit_id``.
+
+.. warning::
+   It is not allowed to create a new branch based on the default branch with no commit history,
```

Comment:
`based on the default branch` doesn't need ?
just `It is not allowed to create a new branch with no commit history`